app/views/moderations/index.html.erb

```diff
@@ -80,7 +80,7 @@
       if (el.innerHTML.length > 10) {
```

Comment:
Just curious what this logic means. Consider adding a comment explaining why the `innerHTML` gets cleared out if it's length is > 10.

Reply:
Hmm I think this was written by someone else but I _think_ this is to clear the innerHTML if the iframe is loaded. Will add a comment here.